WooCommerce/WooCommerceScreenshots/Screens/Login/WelcomeScreen.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+import XCTest
+
+class WelcomeScreen: BaseScreen {
+
+    private struct ElementStringIDs {
+        static let loginButton = "Prologue Log In Button"
+    }
+
+    let logInButton: XCUIElement
+
+    init() {
+        logInButton = XCUIApplication().buttons.firstMatch
```

Comment:
Is there a reason why you used the first match instead of the ID string here? I'd prefer to use an accessibility ID so we're confirming that the exact button we expect (not just any button) is present here.

Reply:
Good question! It's fixed now 😄